### PR TITLE
feat: add Swift Content Card

### LINKED ISSUE #777: Swift card / Content Card
Please create a new content card to feature Swift product image and short text. Please see design in Figma. 

Please see figma link with required info (this is instead of Zeplin)
https://www.figma.com/file/rQRpNn6RM6nUcbm2HCWQlO/WMN-Design-System?node-id=222%3A1752

This is how it is going to be used: 
https://www.figma.com/file/PDLwmCZgpcrI6hfIHKYLku/New-Swift-Custom-Pages?node-id=13%3A12439

Documentation:

**What does it do?**
Shows the image of Swift card and explains how to purchase it

**When to use it?**
Only on Swift product landing pages such as Swift Go and PAYG

**When not to use it?**
On informational content pages. If you need to use a content card use a standard content card pattern.

**How it works**
- User can add Swift card image
- User can edit tile
- User can add description with bullet points and links
- This card is not clickable

### PR DESCRIPTION
**Pull request by houbly**
fixes #777

### CODE REVIEW

File: src/www/pages/patterns/content-cards/index.njk.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+{% from "wmnds/patterns/content-cards/swift-card/_swift-card.njk" import wmndsContentCardSwiftCard %}
+{% markdown %}
+{# Swift card #}
+
+## Swift card
+
+<h3>What does it do?</h3>
+
+Shows the image of Swift card and explains how to purchase it
```
Comment (gldgrnt) [inline src/www/pages/patterns/content-cards/index.njk.md:269]:
```suggestion
- Shows the image of Swift card and explains how to purchase it
```

File: src/www/pages/patterns/content-cards/index.njk.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+
+<h3>When to use it?</h3>
+
+Only on Swift product landing pages such as Swift Go and PAYG
```
Comment (gldgrnt) [inline src/www/pages/patterns/content-cards/index.njk.md:273]:
```suggestion
- Only on Swift product landing pages such as Swift Go and PAYG
```

**Review by gldgrnt — CHANGES_REQUESTED** [review]
Looks good but is missing the 'When not to use it?' section. Also added bullet point suggestion to keep the content consistent with the rest of the page.

**Review by KaterinaKir — APPROVED** [review]
Looks good to me.

**Review by gldgrnt — APPROVED** [review]
👍🏾

**Comment by mrmjprice** [comment]
https://github.com/wmcadigital/wmn-cms/issues/114

**Pull request merged**